nearlib/signing/account_info.js

```diff
@@ -0,0 +1,79 @@
+const KeyPair = require('./key_pair');
+
+/**
+ * Standard format for storing account info in json
+ */
+class AccountInfo {
+
+    constructor(accountId, keyPair, networkId) {
+        this.accountId = accountId;
+        this.keyPair = keyPair;
+        this.networkId = networkId;
+    }
+
+    /**
+     * Reconstruct account info object from json.
+     * @param {Object} json 
+     */
+    static fromJson(json) {
+        if (!json.public_key || !json.secret_key || !json.account_id || !json.network_id) {
+            throw 'Invalid account info format. Please ensure it contains public_key, secret_key, and account_id".';
+        }
+        return new AccountInfo(json.account_id, new KeyPair(json.public_key, json.secret_key), json.network_id);
+    }
+
+    /**
+     * Convert to standard json format.
+     */
+    toJson() {
```

Comment:
If you rename this to `toJSON`, `JSON.stringify` will call it automatically, see:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify#toJSON()_behavior

Reply:
done